### ISSUE TITLE
Paper reproduction

### ISSUE THREAD

**Issue by xuliuwei** [megvii-research/LBHomo#7]
I recently tried to reproduce the results of your paper, but I was unable to achieve the results of the paper. I trained with two 3090 students and batchsize of 12, and the learning rate is the same as that of the paper. May I ask whether the learning rate is the reason for the poor result? Does the learning rate have a big impact on your result? Can you provide your pre-training model?

**Comment by lhaippp** [comment]
Did you leverage the python environments in this repo?
```
pip install -r requirements.txt
```

**Comment by xuliuwei** [comment]
I'll try to change the environment. Another problem is that I found that the training parameter Settings in the paper are different from the parameter Settings in the code, which one should I use?
![image](https://github.com/megvii-research/LBHomo/assets/56248648/8463eec7-cc87-4318-901e-3faa9777ea7c)
![image](https://github.com/megvii-research/LBHomo/assets/56248648/ff60c259-ca61-436e-92d3-6209045e9e17)

**Comment by lhaippp** [comment]
I recommend you to use the experiment setting in this repo

**Comment by xuliuwei** [comment]
Did you run it with four graphics cards?

**Comment by lhaippp** [comment]
I think the number of GPU cards is not important. As long as you can run the code with the setting

**Comment by xuliuwei** [comment]
How long does it take you to train an epoch? I used two 3090s and it took 3.5 hours to train an epoch.

**Comment by six-finger** [comment]
@xuliuwei  May I chat with you about this issue? Thank you~ Wechat: margin333
